modules/adopter-registration-impl/src/main/java/org/opencastproject/adopter/statistic/ScheduledDataCollector.java

```diff
@@ -224,8 +232,19 @@ public void run() {
 
       if (adopter.allowsStatistics()) {
         try {
-          String statisticDataAsJson = collectStatisticData(adopter.getAdopterKey(), adopter.getStatisticKey());
-          sender.sendStatistics(statisticDataAsJson);
+          StatisticData statisticData = collectStatisticData(adopter.getAdopterKey(), adopter.getStatisticKey());
+          sender.sendStatistics(statisticData.jsonify());
+          if (null != tobiraEndpoint) {
+            JsonObject tobiraJson = tobiraEndpoint.getStats();
+            // This is null in the case that Tobira hasn't sent any stats yet.
+            // This could be due to Tobira not existing, or because we've just rebooted.
+            if (null != tobiraJson) {
+              JsonObject blob = new Gson().fromJson("{}", JsonElement.class).getAsJsonObject();
```

Comment:
Shouldn't `new JsonObject()` do the same thing? At least the docs I found say that the constructor returns an empty object.

Reply:
Derp, thanks, good catch.